packages/flutter/lib/src/material/dialog.dart

```diff
@@ -126,9 +121,9 @@ class Dialog extends StatelessWidget {
   /// {@macro flutter.widgets.child}
   final Widget child;
 
-  // TODO(johnsonmh): Update default dialog border radius to 4.0 to match material spec.
+  // Is 4.0 still spec? Is this breaking?
   static const RoundedRectangleBorder _defaultDialogShape =
-    RoundedRectangleBorder(borderRadius: BorderRadius.all(Radius.circular(2.0)));
+    RoundedRectangleBorder(borderRadius: BorderRadius.all(Radius.circular(4.0)));
```

Comment:
@HansMuller where can I check to see if this is still the spec value? https://material.io/components/dialogs did not specify.